gaia/geo/processes_vector.py

```diff
@@ -26,31 +47,67 @@ class BufferProcess(GaiaProcess):
     required_args = ('buffer_size',)
     default_output = formats.JSON
 
-    def __init__(self, **kwargs):
-        super(BufferProcess, self).__init__(**kwargs)
+    def __init__(self, inputs=None, buffer_size=None, **kwargs):
+        super(BufferProcess, self).__init__(inputs, **kwargs)
+        self.buffer_size = buffer_size
         if not self.output:
             self.output = VectorFileIO(name='result',
                                        uri=self.get_outpath())
+        self.validate()
 
-    def compute(self):
-        original_projection = self.inputs[0].get_epsg()
+    def calc_pandas(self):
```

Comment:
@mbertrand we have to find a better way of separating panadas implementation and postgis implementation.